foreman-selinux-enable

```diff
@@ -17,20 +18,22 @@ is_redhat_6() {
 for selinuxvariant in targeted
 do
   if /usr/sbin/semodule -s $selinuxvariant -l >/dev/null; then
+    /usr/sbin/semanage port -E > $TMP_PORTS
+
+    # Remove previously defined elasticsearch_port_t
+    # (this can be removed in future release)
+    grep elasticsearch_port_t $TMP_PORTS | sed s/-a/-d/g >> $TMP
```

Comment:
I think this needs to be executed _before_ the policy is loaded, otherwise the port can't be removed:

<pre>
+ for selinuxvariant in targeted
+ /usr/sbin/semodule -s targeted -l
+ /usr/sbin/semanage port -E
+ sed s/-a/-d/g
+ grep elasticsearch_port_t /tmp/foreman-selinux-enable.r7p4qWmmp9
+ /usr/sbin/semanage module -S targeted -a /usr/share/selinux/targeted/foreman.pp.bz2
libsepol.context_from_record: type elasticsearch_port_t is not defined (No such file or directory).
libsepol.context_from_record: could not create context structure (Invalid argument).
libsepol.port_from_record: could not create port structure for range 9200:9300 (tcp) (Invalid argument).
libsepol.sepol_port_modify: could not load port range 9200 - 9300 (tcp) (Invalid argument).
libsemanage.dbase_policydb_modify: could not modify record value (Invalid argument).
libsemanage.semanage_base_merge_components: could not merge local modifications into policy (Invalid argument).
/usr/sbin/semanage: Could not commit semanage transaction
</pre>